deliverables/ruby_monk_problems/sum_of_cubes.rb

```diff
@@ -1,3 +1,4 @@
 def sum_of_cubes(a, b)
-  # Write your code here
+  (a..b).inject(0) { |sum, x| sum += (x*x*x)}
 end
+# not really sure how .inject works.  Why (0)? 
```

Comment:
Inject is really confusing; it's taken me years to get it.  There are ways to accomplish the same task without inject.  

Here the 0 is the starting value of `sum`.  Each time through the loop, `sum` is set.  First time it is 0, then for the second, third, etc iterations it's set to the *return value of the previous iteration*.